modules/kargoBidAdapter.js

```diff
@@ -4,286 +4,481 @@ import { registerBidder } from '../src/adapters/bidderFactory.js';
-            .replace('{INDEX}', i)
-            .replace('{GDPR}', gdpr)
-            .replace('{GDPR_CONSENT}', gdprConsentString)
-            .replace('{US_PRIVACY}', usPrivacy || '')
-        });
-      }
+
+    const bidResponse = {
+      ad: adUnit.adm,
+      requestId: bidId,
+      cpm: Number(adUnit.cpm),
+      width: adUnit.width,
+      height: adUnit.height,
+      ttl: 300,
+      creativeId: adUnit.id,
+      dealId: adUnit.targetingCustom,
+      netRevenue: true,
+      currency: adUnit.currency || bidRequest.currency,
+      mediaType: meta.mediaType,
+      meta: meta
+    };
+
+    if (meta.mediaType == VIDEO) {
+      bidResponse.vastXml = adUnit.adm;
     }
+
+    bidResponses.push(bidResponse);
+  }
+
+  return bidResponses;
+}
+
+function getUserSyncs(syncOptions, responses, gdprConsent, usPrivacy) {
+  const syncs = [];
+  const seed = spec._generateRandomUuid();
+  const clientId = spec._getClientId();
+  var gdpr = (gdprConsent && gdprConsent.gdprApplies) ? 1 : 0;
+  var gdprConsentString = (gdprConsent && gdprConsent.consentString) ? gdprConsent.consentString : '';
+  // don't sync if opted out via usPrivacy
+  if (typeof usPrivacy == 'string' && usPrivacy.length == 4 && usPrivacy[0] == 1 && usPrivacy[2] == 'Y') {
     return syncs;
-  },
-  supportedMediaTypes: SUPPORTED_MEDIA_TYPES,
-  onTimeout: function(timeoutData) {
-    if (timeoutData == null) {
-      return;
+  }
+  if (syncOptions.iframeEnabled && seed && clientId) {
+    for (let i = 0; i < CERBERUS.SYNC_COUNT; i++) {
+      syncs.push({
+        type: 'iframe',
+        url: CERBERUS.SYNC_URL.replace('{UUID}', clientId)
+          .replace('{SEED}', seed)
+          .replace('{INDEX}', i)
+          .replace('{GDPR}', gdpr)
+          .replace('{GDPR_CONSENT}', gdprConsentString)
+          .replace('{US_PRIVACY}', usPrivacy || '')
+      });
     }
+  }
+  return syncs;
+}
 
-    timeoutData.forEach((bid) => {
-      this._sendTimeoutData(bid.auctionId, bid.timeout);
-    });
-  },
-
-  _getCrbFromCookie() {
-    try {
-      const crb = JSON.parse(storage.getCookie('krg_crb'));
-      if (crb && crb.v) {
-        let vParsed = JSON.parse(atob(crb.v));
-        if (vParsed) {
-          return vParsed;
-        }
+function onTimeout(timeoutData) {
+  if (timeoutData == null) {
+    return;
+  }
+
+  timeoutData.forEach((bid) => {
+    this._sendTimeoutData(bid.auctionId, bid.timeout);
+  });
+}
+
+function _getCrbFromCookie() {
+  try {
+    const crb = JSON.parse(STORAGE.getCookie(CERBERUS.KEY));
+    if (crb && crb.v) {
+      let vParsed = JSON.parse(atob(crb.v));
+      if (vParsed) {
+        return vParsed;
       }
-      return {};
-    } catch (e) {
-      return {};
     }
-  },
+    return {};
+  } catch (e) {
+    return {};
+  }
+}
 
-  _getCrbFromLocalStorage() {
-    try {
-      return JSON.parse(atob(spec._getLocalStorageSafely('krg_crb')));
-    } catch (e) {
-      return {};
-    }
-  },
+function _getCrbFromLocalStorage() {
```

Comment:
Now that these are functions declared in the file, can probably omit the usages of `_<functionName>`.
```suggestion
function getCrbFromLocalStorage() {
```

Reply:
I have done this for all non-exported functions. Functions that are exported in 'spec' object that are not expected still carry this prefix to distinguish them. Happy to remove from all if you think so